### PR TITLE
Regular updates (May 7) and plot updates

### PR DESCRIPTION
**Pull request by jorgeorpinel**
- Fix plenty typos in recent metrics/plots cmd refs 😓 - in 29a8f3c
- Terminology updates (part of #608) - following commits

### CODE REVIEW

File: content/docs/command-reference/plot/diff.md [hunk 1]
```diff
@@ -105,8 +105,9 @@ file:///Users/dmitry/src/plot/logs.csv.html
 
 ![](/img/plot_diff.svg)
 
-The predefined confusion matrix template shows how continuous metrics difference
-can be faceted by separate plots:
+The predefined confusion matrix (`.dvc/plot/confusion_matrix.json`)
+[template](/doc/command-reference/plot#plot-templates) shows how metric
+differences can be faceted by separate plots:
```
Comment (jorgeorpinel) [inline content/docs/command-reference/plot/diff.md:110]: Hi @dmpetrov can you explain what you meant here? I don't really understand the idea. What does faceted mean in this context? Thanks!

File: content/docs/use-cases/versioning-data-and-model-files.md [hunk 2]
```diff
@@ -69,8 +69,8 @@ There are two ways to get to the previous version of the dataset or model: a
 full <abbr>workspace</abbr> checkout, or checkout of a specific data or model
 file. Let's consider the full checkout first. It's quite straightforward:
 
-> `v1.0` below is a Git tag that should be created in advance to identify the
-> dataset version you are interested in. Any
+> `v1.0` below is a hypothetical Git tag that identifies the dataset version you
```
Comment (shcheklein) [inline content/docs/use-cases/versioning-data-and-model-files.md:72]: `hypothetical` does not sound right to me to be honest

File: content/docs/user-guide/contributing/blog.md [hunk 3]
```diff
@@ -71,9 +71,9 @@ It's recommended to follow all the relevant recommendations from the
 > online image optimizer. Huge images pollute Git index, slow down `git clone`,
 > deployment, and other operations with the repository.
 
-The original image should be 2x height and width of the one you want to see on
-the screen to look sharp. To add images, put them into `static/uploads` and
-reference like this:
+The original image should be 2 x the height and width of the size you want to
```
Comment (shcheklein) [inline content/docs/user-guide/contributing/blog.md:74]: 2x is pretty well defined term I think

File: content/docs/use-cases/versioning-data-and-model-files.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/versioning-data-and-model-files.md:72]: Removed.

File: content/docs/user-guide/contributing/blog.md (hunk 3 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/contributing/blog.md]: Reverted.

**Comment by jorgeorpinel** [comment]
Ok @shcheklein please just check my last commit (https://github.com/iterative/dvc.org/pull/1263/commits/065f78f20ac64ceb75dbdb3b8f7fe1ad6a4a40f1) hich is pretty big, sorry. Preparing for @dmpetrov's next round on metrics/plots which I believe he intends to start this coming week.

**Comment by jorgeorpinel** [comment]
We're ignoring the link checker interpreting md image paths as links right?

image
> https://circleci.com/gh/iterative/dvc.org/3397

**Comment by jorgeorpinel** [comment]
p.s. @dmpetrov: you may want to check the changes I made in the `dvc plot` ref and subcommands (mainly in the examples): https://dvc-landing-2020-05-07-zivvdoo.herokuapp.com/doc/command-reference/plot

**Pull request merged**